bertopic/backend/_spacy.py

```diff
@@ -59,6 +59,12 @@ def __init__(self, embedding_model):
             raise ValueError("Please select a correct Spacy model by either using a string such as 'en_core_web_md' "
                              "or create a nlp model using: `nlp = spacy.load('en_core_web_md')")
 
+    def _get_if_cupy(embedding):
+        """Convert to numpy arrays depending on whether cupy was used or not"""
+        if (method := getattr(embedding, 'get', None) and callable(method):
```

Comment:
Another thought I had was to check for the `asnumpy` method: https://docs.cupy.dev/en/stable/reference/generated/cupy.asnumpy.html - do you have a preference? We could also just check with something like:

```python
if not isinstance(embedding, np.ndarray):
```

Reply:
I would prefer an `isinstance` to prevent python 3.7 not being supported anymore. Also, I do not think this function is necessary (see code below).